### PR TITLE
EWPP-483: Make publication contact field composite and prevent organisation contact revisions from being created.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
## EWPP-483

Make publication contact field composite and prevent organisation contact revisions from being created.
### Change log

- Added:
- Changed: Make publication contact field composite and prevent organisation contact revisions from being created.
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by yenyasinn — DISMISSED** [review]

File: modules/oe_content_entity/modules/oe_content_entity_contact/config/post_updates/00007_node_reference_field/field.storage.oe_contact.oe_node_reference.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@ langcode: en
 status: true
```
Comment (yenyasinn) [inline modules/oe_content_entity/modules/oe_content_entity_contact/config/post_updates/00007_node_reference_field/field.storage.oe_contact.oe_node_reference.yml:2]: `  \Drupal::service('module_installer')->install(['entity_reference_revisions']);` can be removed from `oe_content_entity_contact_post_update_00007()`

Comment (imanoleguskiza) [reply modules/oe_content_entity/modules/oe_content_entity_contact/config/post_updates/00007_node_reference_field/field.storage.oe_contact.oe_node_reference.yml:2]: Good catch.

File: modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.post_update.php [hunk 2]
```diff
@@ -108,7 +108,6 @@ function oe_content_entity_contact_post_update_00006(): void {
  * Create the Contact view mode and reference field storage.
  */
 function oe_content_entity_contact_post_update_00007(): void {
-  \Drupal::service('module_installer')->install(['entity_reference_revisions']);
```
Comment (ademarco) [inline modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.post_update.php:111]: This module is still listed as a dependency and it's in the README, we should require the entity_reference and enable it too in this update hook, I think.

image

**Review by ademarco — CHANGES_REQUESTED** [review]

File: modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.info.yml [hunk 3]
```diff
@@ -9,7 +9,6 @@ dependencies:
   - oe_content:oe_content_featured_media_field
   - oe_media:oe_media
   - address:address
-  - entity_reference_revisions:entity_reference_revisions
```
Comment (yenyasinn) [inline modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.info.yml:12]: `- drupal:entity_reference`

**Review by nagyad — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**